OpenEdXMobile/gradle_scripts/AndroidHelper.gradle

```diff
@@ -0,0 +1,117 @@
+import groovy.json.JsonBuilder
+
+class AndroidHelper {
+
+    static def saveProcessedConfig(project, config) {
+        def path = project.file("assets/config")
+        path.mkdirs()
+        def jsonWriter = new FileWriter(path.absolutePath + "/config.json")
+        def builder = new JsonBuilder(config)
+        jsonWriter.withWriter {
+            builder.writeTo(it)
+        }
+    }
+
+    static def saveGoogleServiceJson(project, firebaseConfig) {
+        def googleServiceJsonPath = project.projectDir.toString() + "/google-services.json"
+        new FileWriter(googleServiceJsonPath).withWriter {
+            getGoogleServicesContent(firebaseConfig).writeTo(it)
+        }
+    }
+
+    static def getGoogleServicesContent(firebaseConfig) {
+        if (checkRequiredFields(firebaseConfig)) {
+            def jsonBuilder = new JsonBuilder()
+            def projectNumber = firebaseConfig.get("PROJECT_NUMBER")
+            def projectId = firebaseConfig.get("PROJECT_ID")
+            def oauthClientId = firebaseConfig.get("OAUTH_CLIENT_ID")
+            def currentKey = firebaseConfig.get("CURRENT_KEY")
+            def otherPlatformOauthClientId = firebaseConfig.get("OTHER_PLATFORM_OAUTH_CLIENT_ID")
+            def appStoreId = firebaseConfig.get("APP_STORE_ID")
+            // construct the google-services.json fields
+            jsonBuilder {
+                project_info {
+                    project_number projectNumber
+                    firebase_url "https://$projectId" + ".firebaseio.com"
+                    project_id projectId
+                    storage_bucket projectId + ".appspot.com"
+                }
+                client([{
+                            client_info {
+                                "mobilesdk_app_id" "1:$projectNumber:android:c04089bb49270266"
```

Comment:
We should avoid the `mobilesdk_app_id` value hard coding

Reply:
done.